src/tools/ts-generator/main.cpp

```diff
@@ -80,34 +75,15 @@ int main(int argc, char* argv[])
     if (! checkOptions(settings))
         return 1;
 
+    // ============ THERMAL : Getting data for generating time-series =========
     auto study = std::make_shared<Data::Study>(true);
     Data::StudyLoadOptions studyOptions;
-    studyOptions.prepareOutput = true;
-
     if (!study->loadFromFolder(settings.studyFolder, studyOptions))
     {
         logs.error() << "Invalid study given to the generator";
         return 1;
     }
 
-    study->initializeRuntimeInfos();
```

Comment:
`study->initializeRuntimeInfos();` : disappeared.
We no longer need to manipulate the **RunTimeInfos** stuff only to reach the random generator.
A **Mersenne twister** instance is now defined and used explicitly a bit further in this file (see below).